cmd/notation/cert.go

```diff
@@ -179,9 +179,9 @@ func certGenerateTestCommand(opts *certGenerateTestOpts) *cobra.Command {
 
 func addCert(opts *certAddOpts) error {
 	storeType := opts.storeType
-	namedStore := opts.namedStore
+	namedStore := strings.TrimSpace(opts.namedStore)
```

Comment:
what about `storeType`?

Reply:
looks like you check it in `AddCertCore`, but the `errorSlice` would be a slice of same errors. I feel we can break it early.

Reply:
Nice catch! Checking storeType as well. As for AddCertCore, since it's export, will need to check storeType again.